docs/samples/client/csharp/oauth/readme.md

```diff
@@ -71,9 +71,12 @@ public class ClientCredentialTokenProvider : AuthenticationTokenProvider
 
     public override GetTokenOptions CreateTokenOptions(IReadOnlyDictionary<string, object> properties)
     {
-        if (properties.TryGetValue(GetTokenOptions.ScopesPropertyName, out var scopes) && scopes is string[] scopeArray &&
-            properties.TryGetValue(GetTokenOptions.TokenUrlPropertyName, out var tokenUri) && tokenUri is string tokenUriValue &&
-            properties.TryGetValue(GetTokenOptions.RefreshUrlPropertyName, out var refreshUri) && refreshUri is string refreshUriValue)
+        if (properties.TryGetValue(GetTokenOptions.ScopesPropertyName, out var scopes) &&
```

Comment:
Why did we change the formatting here?

Reply:
I ran prettier on the entire readme.md file and other parts of it also updated